switch.yml

```diff
@@ -0,0 +1,22 @@
+---
+# This playbook switch all routes:
+# 1. point the "current" route to the stack that was pointed by the "next" route,
+# 2. point the "previous" route to the stack that was pointed by the "current" route just before we changed it in 1)
+# 3. delete the stack that was pointed by the "previous" route just before we changed it in 2)
+# 4. patch the next route to the init stamp
```

Comment:
Can we delete the "next" route instead if it's not pointing to an available stack?

Reply:
For me it's a static route I prefere do not delete the route.